flink-ml-lib/src/main/java/org/apache/flink/ml/common/util/QuantileSummary.java

```diff
@@ -367,6 +373,26 @@ public double getRelativeError() {
         return relativeError;
     }
 
+    public int getCompressThreshold() {
```

Comment:
Is this needed to make this class POJO? Would it be simpler to still keep these member variables `public`?

Reply:
No, it is for unittests currently. But I didn't annotate it with `VisibleForTesting` because I think it is reasonable for users to read the information of a `QuantileSummary`.